.github/workflows/windows.yml

```diff
@@ -402,8 +402,7 @@ jobs:
 
```

Comment:
looks like the `name` is wrong, should be `Python ONNX operators tests`

let's fix separately